src/test/random_tests.cpp

```diff
@@ -40,9 +40,9 @@ BOOST_AUTO_TEST_CASE(fastrandom_tests)
     // Check that a nondeterministic ones are not
     FastRandomContext ctx3;
     FastRandomContext ctx4;
+    BOOST_CHECK(ctx3.randbytes(7) != ctx4.randbytes(7));
```

Comment:
Should the `randbytes`, `rand256` and `rand256` tests each get their own pair of `FastRandomContext`:s to properly test for non-determinism? That way the ordering of the test won't matter.

Reply:
Good idea, done.